CoqBB5/BB5/README.md

```diff
@@ -0,0 +1,268 @@
+# BB(5) = 47,176,870
+
+This folder contains the Coq ([v8.20.1](https://github.com/coq/coq/blob/V8.20.1/INSTALL.md)) proof that `BB(5) = 47,176,870`. This result was not previously known.
+
+This result means that the maximum number of steps that a halting 5-state Turing machine can do from all-0 tape is 47,176,870. See [bbchallenge's wiki](https://wiki.bbchallenge.org/wiki/Main_Page) or [bbchallenge's BB5 paper](https://github.com/bbchallenge/bbchallenge-paper) for more background and detailed information.
+
+Proving this results involves enumerating 5-state Turing machines and deciding for each whether it halts or not and, if it halts, that it halts in at most 47,176,870 steps.
+
+The extracted data from this proof is available at [https://docs.bbchallenge.org/CoqBB5_release_v1.0.0/BB5_verified_enumeration.csv.gz](https://docs.bbchallenge.org/CoqBB5_release_v1.0.0/BB5_verified_enumeration.csv.gz) in the form of a CSV file (**10 Gb**) listing each enumerated machine with its halting status (halt/nonhalt) as well as the ID of the decider that decided it (IDs as defined in `BB5_Deciders_Generic.v`). More details [below](#extracting-results).
+
+## Compiling the proof
+
+Assuming you have [opam installed](https://opam.ocaml.org/doc/Install.html), you can install Coq v8.20.1 using:
+
+```
+opam switch 4.14.1
```

Comment:
It looks like you have to run `opam switch create 4.14.1` if you haven't already installed that version.

Reply:
Fixed